cypress/utils/utils.ts

```diff
@@ -114,35 +114,38 @@ export function cancelForm(): void {
 }
 
 export function login(username?, password?: string): void {
-    cy.visit(tackleUiUrl, { timeout: 120 * SEC });
-    cy.wait(5000);
-    cy.get("h1", { timeout: 120 * SEC }).then(($b) => {
-        if ($b.text().toString().trim() == "Sign in to your account") {
-            if (username && password) {
-                userName = username;
-                userPassword = password;
-            } else {
-                userName = Cypress.env("user");
-                userPassword = Cypress.env("pass");
-            }
-            inputText(loginView.userNameInput, userName);
-            inputText(loginView.userPasswordInput, userPassword);
-            click(loginView.loginButton);
-            // Change default password on first login.
-            cy.get("span").then(($inputErr) => {
-                if ($inputErr.text().toString().trim() == "Invalid username or password.") {
-                    inputText(loginView.userPasswordInput, "Passw0rd!");
-                    click(loginView.loginButton);
-                    updatePassword();
+    cy.session({ username, password }, () => {
+        cy.visit(tackleUiUrl, { timeout: 120 * SEC });
+        // cy.wait(2 * SEC);
```

Comment:
**nitpick:** I think we can safely remove this commented coded and add it later if its needed